tomviz/DataSource.cxx

```diff
@@ -894,6 +909,11 @@ vtkImageData* DataSource::transferFunction2D() const
   return this->Internals->m_transfer2D;
 }
 
+vtkRectd* DataSource::transferFunction2DBox() const
```

Comment:
Why a pointer, rather than return by value?

Reply:
Because the widget needs to be able to update it the same way it updates the vtkColorTransferFunction*.